Description: Use x-terminal-emulator instead of xterm to be more desktop
 independent
Author: Levu <levu@phicode.de>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/282972
Bug-Debian: http://bugs.debian.org/670510
Index: geany/src/keyfile.c
===================================================================
--- geany.orig/src/keyfile.c	2021-10-18 11:24:13.032852456 +0800
+++ geany/src/keyfile.c	2021-10-18 11:24:13.028852393 +0800
@@ -79,7 +79,7 @@
 #elif defined(__APPLE__)
 #define GEANY_DEFAULT_TOOLS_TERMINAL	"open -a terminal %c"
 #else
-#define GEANY_DEFAULT_TOOLS_TERMINAL	"xterm -e \"/bin/sh %c\""
+#define GEANY_DEFAULT_TOOLS_TERMINAL	"x-terminal-emulator -e \"/bin/sh %c\""
 #endif
 #ifdef __APPLE__
 #define GEANY_DEFAULT_TOOLS_BROWSER		"open -a safari"
